From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jakub Jelinek <jakub@redhat.com>
Date: Mon, 1 May 2006 08:02:53 +0000
Subject: [PATCH] Enable SETENT_BATCH_READ nis/nss option by default

* Mon May  1 2006 Jakub Jelinek <jakub@redhat.com> 2.4.90-4
- SETENT_BATCH_READ /etc/default/nss option for speeding up
  some usages of NIS+ (#188246)

diff --git a/nis/nss b/nis/nss
index 0ac6774a1ff29f012efaec9c4be1fcc3b83da7e8..d720e719267db5f741b67e7b98e4052e503c4333 100644
--- a/nis/nss
+++ b/nis/nss
@@ -25,7 +25,7 @@
 #  memory with every getXXent() call.  Otherwise each getXXent() call
 #  might result into a network communication with the server to get
 #  the next entry.
-#SETENT_BATCH_READ=TRUE
+SETENT_BATCH_READ=TRUE
 #
 # ADJUNCT_AS_SHADOW
 #  If set to TRUE, the passwd routines in the NIS NSS module will not
